src/github.com/getlantern/flashlight/analytics/analytics.go

```diff
@@ -33,51 +32,40 @@ var (
 
 	maxWaitForIP = math.MaxInt32 * time.Second
 
-	// We get the user agent to use from live data on the proxy, but don't wait
-	// for it forever!
-	maxWaitForUserAgent = 30 * time.Second
-
-	// This allows us to report a real user agent from clients we see on the
-	// proxy.
-	userAgent = eventual.NewValue()
-
 	// Hash of the executable
 	hash = getExecutableHash()
 )
 
 // Start starts the GA session with the given data.
 func Start(deviceID, version string) func() {
-	return start(deviceID, version, geolookup.GetIP, maxWaitForUserAgent, trackSession)
+	return start(deviceID, version, geolookup.GetIP, trackSession)
 }
 
 // start starts the GA session with the given data.
-func start(deviceID, version string, ipFunc func(time.Duration) string, uaWait time.Duration,
+func start(deviceID, version string, ipFunc func(time.Duration) string,
 	transport func(string, eventual.Getter)) func() {
 	var addr atomic.Value
 	go func() {
-		logging.AddUserAgentListener(func(agent string) {
```

Comment:
@myleshorton With this change, the AddUserAgentListener capability is no longer used.  Can you remove it and the corresponding test please?